src/main/scala/is/hail/methods/Nirvana.scala

```diff
@@ -0,0 +1,351 @@
+package is.hail.methods
+
+import java.io.{FileInputStream, IOException}
+import java.util.Properties
+
+import is.hail.annotations.{Annotation, Querier}
+import is.hail.expr.{JSONAnnotationImpex, Parser, TArray, TBoolean, TDouble, TInt, TSet, TString, TStruct, Type}
+import is.hail.utils._
+import is.hail.variant.{Variant, VariantDataset}
+import org.apache.spark.storage.StorageLevel
+import org.json4s.jackson.JsonMethods
+
+import scala.collection.JavaConverters._
+
+
+object Nirvana {
+
+  // Originally the schema exactly matched Nirvana's JSON output, but in the interest of
```

Comment:
As we're only leaving out three doubles and two ints, nothing compared to the rest, I think leaving them in will cause less confusion for others who are used to running against Nirvana's published schema.